FordLabs/PeopleMover
# Show unsupported browser page when using IE, Edge, or Safari

## Issue
Resolves #503 

## What was done
- [x] Display unsupported browser page when using IE, edge, or safari

## How to test
1. Verify that the unsupported browser page is displayed when using IE, edge, or safari

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
